src/frequenz/channels/_anycast.py

```diff
@@ -68,10 +68,19 @@ def __init__(self, maxsize: int = 10) -> None:
             maxsize: Size of the channel's buffer.
         """
         self.limit: int = maxsize
```

Comment:
Damn, all of these should be private, or read-only at best. Not for this PR though:

* #186

Reply:
I see. Thanks for creating https://github.com/frequenz-floss/frequenz-channels-python/issues/186